plugin/src/main/groovy/com/novoda/staticanalysis/StaticAnalysisPlugin.groovy

```diff
@@ -20,10 +20,11 @@ class StaticAnalysisPlugin implements Plugin<Project> {
     }
 
     private EvaluateViolationsTask createEvaluateViolationsTask(Project project) {
-        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension)
+        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension, project)
```

Comment:
An extension is just an object, so you have to pass a reference of a project somehow. I don't see how this is odd :(